dcat/index.html

```diff
@@ -675,8 +675,8 @@ <h3>RDF representation</h3>
             The axiomatization of DCAT 2014 used global domain and range constraints for many of the properties defined in the DCAT namespace [[?VOCAB-DCAT-20140116]]. This makes quite strong ontological commitments, some of which are now being reconsidered - see individual issues noted inline below.
         </p-->
 
-        <p>The (revised) DCAT vocabulary is <a href="https://github.com/w3c/dxwg/tree/gh-pages/dcat/rdf/">available in RDF</a>.
-            The primary artefact <a href="https://w3c.github.io/dxwg/dcat/rdf/dcat2.ttl"><code>dcat2.ttl</code></a> is a serialization of the core DCAT vocabulary.
+        <p>The (revised) DCAT vocabulary is <a href="https://www.w3.org/ns/dcat2.rdf">available in RDF</a>.
```

Comment:
hhttps://github.com/w3c/dxwg/tree/gh-pages/dcat/rdf/ was the link at the GitHub directory containing the different RDF DCAT serializations. I guess there is no correspondent directory to point at in the W3C Server. IMO,  the landing page for the DCAT serializations could be considered as the closest equivalent link.
Do any of the other editors (@pwin @davebrowning @andrea-perego @dr-shorthair @agbeltran ) concur ?
 ```suggestion
        <p>The (revised) DCAT vocabulary is <a href="https://www.w3.org/ns/dcat.html">available in RDF</a>.
```

Reply:
Seems reasonable

Reply:
It seems we are converging on Andrea's proposal. So we can resolve this and accept the other.